falcon/request.py

```diff
@@ -514,6 +516,35 @@ def cookies(self):
 
         return self._cookies.copy()
 
+    @property
+    def access_route(self):
+        """If a forwarded header exists this is a list of all ip addresses
+        from the client ip to the last proxy server.
+
+        Inspired by werkzeug's access_route
+        """
+        if self._cached_access_route is None:
+            access_route = []
+            if 'HTTP_FORWARDED' in self.env:
+                access_route = self._parse_rfc_forwarded()
+            if not access_route and 'HTTP_X_FORWARDED_FOR' in self.env:
+                # we don't handle X-Fowarded-By since it is not possible to
+                # know in which order the already existing fields were added
+                # just as rfc7239 stated
+                access_route = [ip.strip() for ip in
+                                self.env['HTTP_X_FORWARDED_FOR'].split(',')]
+            if not access_route and 'HTTP_X_REAL_IP' in self.env:
+                access_route = [self.env['HTTP_X_REAL_IP']]
+            if not access_route and 'REMOTE_ADDR' in self.env:
+                access_route = [self.env['REMOTE_ADDR']]
+            self._cached_access_route = access_route
+        return self._cached_access_route
+
+    @property
+    def remote_addr(self):
+        """The remote address of the client."""
```

Comment:
I think the reader may be confused about what this method does and when to use it vs. `access_route`. Would you mind updating the docstring to make this more clear?